inference-engine/tests/functional/plugin/gpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -9,5 +9,8 @@
 
 std::vector<std::string> disabledTestPatterns() {
     return {
+        // TODO: Issue: 32521
+        R"(.*(EltwiseLayerTest).*secondaryInputType=CONSTANT.*netPRC=FP16.*)",
+        R"(.*(EltwiseLayerTest).*IS=.*1.1.1.1.*opType=SCALAR.*)"
```

Comment:
please add JIRA ticket number

Reply:
check if these are valid fails when eltwise.cpp is fixed, also amke sure that these are precise enough if I recall it is failing cause broadcast blobs are limited, so there is chance that it will pass on Prod or/and Parameter as secondary input